charts/trino/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.13.0
+version: 0.13.0-radek
```

Comment:
revert

Reply:
(I mean bump to 0.14.0)

Reply:
To not force user to define `shareProcessNamespace`